projects/aws-containers/hello-eks-anywhere/conf/default.conf

```diff
@@ -0,0 +1,24 @@
+map $http_user_agent $root {
+	"~*curl*" /usr/share/nginx/txt;
+	"~*wget*" /usr/share/nginx/txt;
+	default /usr/share/nginx/html;
+}
+server {
+    listen       80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
```

Comment:
Maybe remove access_log if not used?

Reply:
> There a couple of empty files in the diff: projects/aws-containers/hello-eks-anywhere/conf/index.txt, and another one. Can they be removed?

Yea I'll remove that.

Reply:
For editing the actual files such as the access_log, I prefer not to change them, since they are from another upstream repo, so changing them would require us to maintain the changes. 

https://github.com/aws-containers/hello-eks-anywhere